ci/playbooks/tcib/tcib.yml

```diff
@@ -26,23 +26,15 @@
 
     - name: Build tcib package
       vars:
-        cifmw_bop_openstack_project_path: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/tcib"
         cifmw_bop_yum_repos_dir: "{{ cifmw_build_containers_repo_dir }}"
+        cifmw_bop_gating_repo: "{{ cifmw_build_containers_repo_dir }}"
```

Comment:
silly question, but it's my first time looking at this file, what does `bop` stand for?

Reply:
scrolling down I see it's probably short for  `build_openstack_packages` :)